### PR TITLE
Added travis.yml

### PR DESCRIPTION
**Pull request by jroper**

### CODE REVIEW

**Comment by ktoso** [comment]
Looks good, thanks for this James :)

**Comment by jroper** [comment]
We're green now.

**Comment by ktoso** [comment]
Merged, thanks @jroper!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: scala
+
+jdk: oraclejdk8
+
+script:
+  - sbt test activatorRunTutorial activatorGenerateTutorial
+  - git diff --quiet tutorial/index.html || (echo "index.html has been updated directly, do not do this, edit tutorial/index.html, then use https://github.com/typesafehub/activator-tutorial-generator to generate the new index.html" && false)
+
+before_script:
+  - printf "resolvers += \"Typesafe repository\" at \"http://repo.typesafe.com/typesae/releases/\"\n\naddSbtPlugin(\"com.typesafe.sbt\" %%%% \"sbt-activator-tutorial-generator\" %% \"1.0.3\")" > project/activator-tutorial-generator.sbt
```
Comment (ktoso) [inline .travis.yml]: whoops, `typesae/releases` => `typesafe/releases`

Comment (ktoso) [inline .travis.yml]: fixed https://github.com/typesafehub/ReactiveMaps/commit/a02a12c9b0b9d294d35edb5b51cc9a6fc65c6373

**Pull request merged**